paper/config/plugins/SimpleAdminHacks/config.yml

```diff
@@ -454,20 +454,29 @@ hacks:
-       - "   &1RealisticBiomes&r\nYou will starve! Crops and trees require sunlight or glowstone, grow slowly across the board, and do not grow in inappropriate biomes. This encourages trade and warfare because diverse crops are required for FactoryMod XP production."
-       - "   &5Closing Remarks&r\nType /help for more info on everything. Go to https://redd.it/j3nk2f for a more detailed guide. There is much more to learn."
-       - "If you would like to report hacks or glitches, please message the admins through subreddit modmail at reddit.com/r/CivMC\n\nDisclaimer: &2TangentialThreat&r is not staff and should not be trusted."
+       - "&5Hello and welcome to CivMC, &2${player}!\n\n   A server based on nation building and player freedom. Where admin control is minimal, and players dispense their own justice." 
+	   	   - "   &1Namelayer&r\nPocket radios! Create or join chat groups to communicate with friends all over CivMC. By default you have joined global chat or ! chat. You can switch from local chat (1000 block radius) to this one by performing the command: /g !"
+	   - "Go back to localchat by performing the /g command on it own. This works for any group you are allowed to speak in. just switch ! for the name of the group. \nCreate your own group using the GUI by perfoming the /nl Command."
+	  	   - "   &1Citadel&r\nAnother use for your Namelayer groups! Slow down raiders by reinforcing blocks such as your house or your chests and doors to lock them. Reinforced blocks take multiple breaks to drop for players not on the namelayer group."
+	   - "To reinforce a block perform the command /ctr (Groupname) while holding stone, iron or diamond. then punch any block to reinforce it to that group. Now only your allies have ease of access! Tip: Use /ctf instead to reinforce while building!"
+ 	   - "   &1Hiddenore&r\nSudden iron surrounds you! Ores spawn around you as you mine. While caves are still dusted with some visible ores, the main way to get ores is to dig. Ores still hold the same height level for their rarity.
+	   - "Diamonds spawn in veins, where you find one, you'll find others. A message will appear in chat when you've found one. Use silk touch on all ores for increased returns when smelting them in the Ore smelter factory. (better than fortune!)"
+	   - "   &1JukeAlert&r\nSmile! you're on snitches! Jukeboxes and noteblocks when reinforced will turn into a snitch and notify players on the group when other players enter its 21x21 sphere. Noteblocks turned snitch only hold this one function." 
+	   - "jukeboxes in addition also log some actions, like block breaks or the opening of doors and chests. You can access a jukebox and it's logs if you're on its group by performing the /ja command. To name a nearby snitch perform the /janame (Snitchname)"
```

Comment:
- noteblock -> note-block
- 21x21 sphere -> 21x21 bounding box
- Capitalize Jukebox on L467.
- it's logs -> its logs
- `/janame Snitchname` -> `/janame <snitchname>`

Reply:
I can't remember which JA or Bastion measure from the center or don't include the center in the radius but I know they both do it differently, but I'm not sure which is 21x21 and which is 23x23

Reply:
Please someone confirm what the actual size is. I always get them confused.